LuckyDragon/app/src/main/java/com/example/luckydragon/Fragments/OrganizerEventFragment.java

```diff
@@ -74,6 +80,26 @@ public void updateWaitlistCapacity() {
         waitlistCapacityTextView.setText(String.format("Capacity: %s", waitlistLimit));
     }
 
+    /**
+     * Displays inviteelist textview or inviteelist listview
+     */
+    public void displayInviteelist() {
+        TextView noinviteesTextView = getView().findViewById(R.id.noinviteesTextView);
+        if (event.getInviteelistUsers().size() == 0) {
+            noinviteesTextView.setVisibility(View.VISIBLE);
+        }
+    }
+
+    /**
+     * Displays waitlist textview or waitlist listview
+     */
+    public void displayWaitlist() {
+        TextView noWaitlisteesTextView = getView().findViewById(R.id.noWaitlisteesTextView);
+        if (event.getWaitlistUsers().size() == 0) {
+            noWaitlisteesTextView.setVisibility(View.VISIBLE);
+        }
+    }
```

Comment:
No logic should be in fragment ie try to make it work like this:
```suggestion
    public void hideWaitlist() {
            TextView noWaitlisteesTextView = getView().findViewById(R.id.noWaitlisteesTextView);
            noWaitlisteesTextView.setVisibility(View.VISIBLE);
    }
```